### ISSUE TITLE
Game shows broken menu on launch

### ISSUE THREAD

**Issue by garrettburnett** [timkurvers/valheim-macos#81]
Hey there, super grateful for all the work done here. 

I'm able to get the .app binary built correctly, but when I run it this is what I see after the intro logo animations...

Seems like none of the renderers are loading correctly.  

Any idea?

<img width="1063" alt="image" src="https://github.com/timkurvers/valheim-macos/assets/14268490/77ae1c7c-3979-4f32-837f-602f3d021a42">

**Comment by timkurvers** [comment]
That's a funky looking window 😅

- What are your hardware specs?
- Which macOS version?
- Anything of note in `~/Library/Logs/IronGate/Valheim/Player.log`?
   <super>(be a bit careful sharing this file, as it may contain personal details)</super>

**Comment by garrettburnett** [comment]
Hey there--I knew I was forgetting something obvious...
13-Inch Macbook Pro, M1 (2020)
MacOS 14.2.1
Memory 8GB

This seems to be the primary issue:
```
OPENGL LOG: Creating OpenGL 4.1 graphics device ; Context level  <OpenGL 4.1> ; Context handle -285077504
Begin MonoManager ReloadAssembly
- Loaded All Assemblies, in  0.255 seconds
- Finished resetting the current domain, in  0.004 seconds
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
WARNING: RGB Compressed BC6H UFloat format is not supported, decompressing texture
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
Unknown error occurred while loading '/Users/Garrett/Documents/valheim-macos/build/Valheim.app/Contents/Resources/Data/sharedassets0.assets'.
```
These messages repeat infinitely while running the game.

**Comment by timkurvers** [comment]
That's very odd indeed 🤔 I do not have any concrete answers here, but two ideas:

- What is the size of the `build` folder? Either check this via Finder, or something ala `du -sh Documents/valheim-macos/build` in a terminal. It should be around 1.4 GB.
- You could try moving the entire project out of the Documents folder, into a folder that may be less restrictive (e.g. `Projects` in your home folder) and then try running the build script again, making a fresh build.
-

**Comment by timkurvers** [comment]
Closing this issue as the official macOS client is now available! 🥳 (see #104)
